src/python/WMCore/WorkQueue/Policy/Start/StartPolicyInterface.py

```diff
@@ -36,10 +34,10 @@ def __init__(self, **args):
         self.badWork = []  # list of bad work unit (e.g. without any valid files)
         self.pileupData = {}
         self.cric = CRIC()
-        if usingRucio():
-            self.rucio = Rucio(self.args['rucioAcct'], configDict={'logger': self.logger})
-        else:
-            self.phedex = PhEDEx()  # this will go away eventually
+        # FIXME: for the moment, it will always use the default value
+        self.rucioAcct = self.args.get("rucioAcct", "wmcore_transferor")
+        if not self.rucio:
+            self.rucio = Rucio(self.rucioAcct, configDict={'logger': self.logger})
```

Comment:
I think this Rucio object is only meant to be created by unit tests (which will use the mocked version). Otherwise the Rucio object will come upstream.
And we definitely want to let a possible exception to propagate upstream here, such that we cancel the workqueue element creation.

Reply:
Ok, I get it. Thanks!